cmd/main.go

```diff
@@ -141,6 +144,8 @@ var Options struct {
 	LivenessValidationTimeout      time.Duration `envconfig:"LIVENESS_VALIDATION_TIMEOUT" default:"5m"`
 	ApproveCsrsRequeueDuration     time.Duration `envconfig:"APPROVE_CSRS_REQUEUE_DURATION" default:"1m"`
 	HTTPListenPort                 string        `envconfig:"HTTP_LISTEN_PORT" default:""`
+	MinimalVersionForConvergedFlow string        `envconfig:"MINIMAL_VERSION_FOR_COVERGED_FLOW" default:"4.11.0"`
```

Comment:
What do you think about making the version a constant and changing this option to a `bool`? 

I don't think we need the version to be configurable. The converged flow will be available starting on 4.11 and deployments can either use it or not.